open-banking-accelerator/components/consent-management/com.wso2.openbanking.accelerator.consent.extensions/src/main/java/com/wso2/openbanking/accelerator/consent/extensions/common/factory/AcceleratorConsentExtensionFactory.java

```diff
@@ -17,14 +17,6 @@
  */
 package com.wso2.openbanking.accelerator.consent.extensions.common.factory;
 
-import com.wso2.openbanking.accelerator.consent.extensions.authorize.impl.handler.persist.AccountConsentPersistenceHandler;
```

Comment:
Let's only remove retrieval related imports here, as we can remove persistance related imports in other PR

Reply:
Fixed in [01f2f08]